brainstormforce/astra
# Fix: "Can't read property querySelector of null" error due to conflict with HFE

### Description
With the Header Footer for Elementor plugin, we can create a site header with this plugin. This custom header overrides the theme header.
In this case, Astra throws an error as we are using a selector without the null check.

### Screenshots
https://share.getcloudapp.com/YEuXKbqj

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
- Install and activate the HFE plugin on your site
- Visit Dashboard -> Appearance -> Header Footer & Blocks 
- Add a new template and assign a type "Header" [Link](https://share.getcloudapp.com/Jru1vZjg)
- Edit with Elementor
- Same error appears in the console

### Checklist:
